immersion/kubeflow_pipelines/pipelines/solutions/lab-02_vertex.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<p>We can remove this as <code>google-cloud-aiplatform=1.1.1</code> is installed in the latest GCP Notebooks. (This is now also true for the earlier lab_01.)</p><p> </p><p>However, we do need to install <code>kfp</code>!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done!</p>

Reply:
<p>Actually, I even removed the dependency on kfp by using the newer way of running pipelines using the aiplatform sdk.</p>